src/main/java/sg/ncl/MainController.java

```diff
@@ -70,6 +70,9 @@
     private static final String CONTACT_EMAIL = "support@ncl.sg";
 
     private static final String UNKNOWN = "?";
+    private static final String MESSAGE = "message";
+    private static final String MESSAGE_SUCCESS = "messageSuccess";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "message" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)